README.rst

```diff
@@ -18,6 +18,30 @@ Installation
 
     conda install -c conda-forge nc_time_axis
 
+
+Example Code
+------------
+::
+
+    import nc_time_axis
+	import netcdftime
+	import matplotlib.pyplot as plt
+	import random
+	DT = [netcdftime.datetime(year=2017, month=2, day=day) for day in range(1, 31)]
+	CDT = [nc_time_axis.CalendarDateTime(item, "360_day") for item in DT]
+	TempList = []
```

Comment:
Wikipedia has an entry about [naming conventions in python](https://en.wikipedia.org/wiki/Naming_convention_%28programming%29#Python_and_Ruby)